internal/servers/worker.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 type workerAuthWorkerId struct {
-	WorkerId string
+	WorkerId string `mapstructure:"worker_id"`
```

Comment:
Can you update the AttachWorkerIdToState function below to reflect this change? `TestStoreWorkerAuth` is failing because it is no longer setting the workerId correctly

Reply:
I have this fixed in another commit now